charts/openebs/README.md

```diff
@@ -95,34 +95,34 @@ The following table lists the common configurable parameters of the OpenEBS char
 | ----------------------------------------| --------------------------------------------- | ----------------------------------------- |
 | `apiserver.enabled`                     | Enable API Server                             | `true`                                    |
 | `apiserver.image`                       | Image for API Server                          | `openebs/m-apiserver`                     |
-| `apiserver.imageTag`                    | Image Tag for API Server                      | `2.12.0`                                  |
+| `apiserver.imageTag`                    | Image Tag for API Server                      | `2.12.2`                                  |
```

Comment:
Isn't m-apiserver not going to be installed for OpenEBS 3.x? If it is getting installed at all, shouldn't it be disabled by default(i.e. `apiserver.enabled=false`). Is it preserved for a few releases to help users migrate to newer CSI based provisioners?

Reply:
Disabled legacy components.